src/components/Login/Login.tsx

```diff
@@ -1,18 +1,132 @@
-import Button from './Button';
-import Input from './Input';
+import api from '@/Api/api';
+import { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { useNavigate } from 'react-router';
+import { Link } from 'react-router-dom';
+import { HOME_PAGE, SIGNUP_PAGE } from '@/utils/constants';
 import Logo from './Logo';
 
 const Login = () => {
+  const navigate = useNavigate();
+  const [account, setAccount] = useState({
+    email: '',
+    password: '',
+  });
+
+  const onChangeInputValue = (e: any) => {
```

Comment:
```suggestion
  const onChangeInputValue = ({target}: React. ChangeEvent< HTMLInputElement >) => {
```
p1;
- any type 대신에 React에서 제공하는 type 사용하면 안전하게 로직을 만들 수 있어요.
- [React Typescript onChange Event Type](https://stackoverflow.com/questions/40676343/typescript-input-onchange-event-target-value)

p5;
- 구조분해 할당 문법도 JS에서 사용할 수 있으니 제안드려요~
- [MDN - 구조 분해 할당](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)

Reply:
수정했습니다~